releases/release-1.31/release-notes/maps/pr-125257-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 125257
+releasenote:
+  text: Graduated Kubernetes' support for AppArmor to GA.
```

Comment:
```suggestion
  text: Graduated Kubernetes' support for AppArmor to GA. You now cannot disable the `AppArmor`
    feature gate.
```
?

Reply:
without "the" ? @sftim

Reply:
If you mean _“You now cannot disable `AppArmor` feature gate.“_, @Checksumz, that wouldn't be idiomatic English.